### DISCUSSION TITLE
Doc comment on Create objects (/manage-data/create)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1846] [Docs/blog comments]
# Doc comment on Create objects (/manage-data/create)

Every object in Weaviate belongs to exactly one collection. Use the examples on this page to manage your collections.

https://weaviate.io/developers/weaviate/manage-data/create

**Comment by xiaoli2021** [comment]
For Create an object with named vectors: I need to use third part model to generate embedding, and I did not define multiple (named) vectors in my collection, and also some of my properties do not need to have embedding array.  how to assign some of the properties with specified vectors ?
